### PR TITLE
Added Query-Id to http response header

### PR DESCRIPTION
**Pull request by fandyushin**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

For changelog. Remove if this is non-significant change.

Category (leave one):
- Improvement

Short description:
Added Query-Id to Server's HTTP Response header

### CODE REVIEW

File: dbms/src/Interpreters/DDLWorker.cpp [hunk 1]
```diff
@@ -528,7 +528,7 @@ bool DDLWorker::tryExecuteQuery(const String & query, const DDLTask & task, Exec
     {
         current_context = std::make_unique<Context>(context);
         current_context->setCurrentQueryId(""); // generate random query_id
-        executeQuery(istr, ostr, false, *current_context, nullptr);
+        executeQuery(istr, ostr, false, *current_context, nullptr, nullptr);
```
Comment (alexey-milovidov) [inline dbms/src/Interpreters/DDLWorker.cpp]: It's strange that there was nullptr here. Can we pass `{}`?

Comment (fandyushin) [reply dbms/src/Interpreters/DDLWorker.cpp]: done

**Comment by alexey-milovidov** [comment]
Why is it named `Query-Id` instead of `X-ClickHouse-Query-Id`?

**Comment by fandyushin** [comment]
> Why is it named `Query-Id` instead of `X-ClickHouse-Query-Id`?

I asked proper name on meetup (https://events.yandex.ru/events/ClickHouse/2-feb-2019/) And this name was chosen. I can fix it, should I use this pr or make new?

**Comment by alexey-milovidov** [comment]
You can make a new PR.

PS. Why I put attention on this?
A user proposed to add `X-ClickHouse-Exception-Code` header. We also have `X-ClickHouse-Progress`. I started to dig into code for quick implementation... and found this inconsistency.

**Pull request merged**